pages/auth/login.tsx

```diff
@@ -1,7 +1,60 @@
 import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useTranslations } from "next-intl";
+import LoginForm from "@/resources/inputs/login-form";
+import register_page_style from "@/styles/pages/register.module.css";
+import { getStaticPropsWithTranslations } from "@/modules/lang/props";
+import { GetStaticProps } from "next";
 
-const Login: React.FC = () => {
-  return <h1>Login Page</h1>;
+export const getStaticProps: GetStaticProps = getStaticPropsWithTranslations;
+
+const LoginPage: React.FC = () => {
+  const router = useRouter();
+  const t = useTranslations("");
+  const handleLoginSuccess = (user: any, token: string) => {
+    console.log("Login successful:", user, token);
+    localStorage.setItem("token", token);
+    localStorage.setItem("user", JSON.stringify(user));
```

Comment:
I dont want `localStorage` in the components,  

This probable need to go in a context, available for all the users. 

local storage shall be access from a single place where other